docs/quick_start.md

```diff
@@ -37,7 +37,7 @@ Honeycomb relies on Electron to package the cross-platform desktop applications.
 Once [Node.js](https://nodejs.org/en/download/) is installed you will able to use `npm` commands in the terminal. To install the dependencies for HoneyComb run the following command at the terminal (remember you need `cd your-new-task-name` before)
 
 ```
-npm install
+npm install -D
```

Comment:
is the `-D` needed?

Reply:
I think dotenv cli, dotenv and crossenv are all in devdependencies and they are not needed for runtime, so I think its needed?

Reply:
I thought the default behavior of `npm install` was to also install dev dependencies - is that not the case?

Reply:
oh yeah it does, I don't know why the docs in MSIT included the -D then.